pkg/logql/syntax/ast.go

```diff
@@ -1337,8 +1337,11 @@ func (e *RangeAggregationExpr) String() string {
 }
 
 // impl SampleExpr
-func (e *RangeAggregationExpr) Shardable() bool {
-	return shardableOps[e.Operation] && e.Left.Shardable()
+func (e *RangeAggregationExpr) Shardable(topLevel bool) bool {
+	if e.Operation == OpRangeTypeQuantile && !topLevel {
+		return false
+	}
```

Comment:
Could you add a comment here as well that we are ignoring the feature flag?

Reply:
The AST code currently doesn't have any knowledge of the feature flag the shardmapper uses, but I can add a comment to clarify the behaviour.